doc/Language/classtut.pod6

```diff
@@ -314,7 +314,7 @@ initialized when being declared, however we are implementing the Singleton
 pattern here and the object must be created during its first use. It is not 100%
 possible to predict the moment when attribute initialization will be executed,
 because it can take place during compilation, runtime or both, especially when
-importing the class using the L<use|/syntax/use> keyword.
+importing the class using the C<use> keyword.
```

Comment:
Why is this link eliminated?

Reply:
Where does it go to? I could not identify a suitable target

Reply:
found appropriate place to target. Reinserted link.